challenges/prototypes.js

```diff
@@ -5,29 +5,45 @@
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
-
+function CuboidMaker(obj) {
+  this.length = obj.length;
+  this.width = obj.width;
+  this.height = obj.height;
+}
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
-  
+ 
   Formula for cuboid volume: length * width * height
 */
 
-
+CuboidMaker.prototype.volume = function() {
+  return this.length * this.height * this.width;
+};
 /* == Step 3: Surface Area Method ==
   Create another method using CuboidMaker's prototype that returns the surface area of a given cuboid's length, width, and height. 
 
   Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
 */
 
-
```

Comment:
make sure you pay attention to the names and what they are asking you to do
CuboidMaker.prototype.surfaceArea = function surfaceArea() {
  return `Surface area: ${2 * (this.length * this.width + this.length * this.height + this.width * this.height)}`;
};

Reply:
you had made 2 volumes, but this one should have been surfaceArea